### ISSUE TITLE
[a11y] On file upload questions, screen reader doesn't read question or help text

### ISSUE THREAD

**Issue by melanie-exygy** [civiform/civiform#10438]
**Describe the bug**
Both in North Star and pre-NS, the screen reader doesn't read the question or the help text on file upload questions.  This is a serious problem because a screen reader user won't know what file to upload, whether it is their driver's license, their housing application, their birth certificate, etc.

**To Reproduce**

1. Create a program with a file upload question.
2. Turn the screen reader on.
3. Apply to the program.
4. When on the page with the file upload question, use the Tab key to tab to the question.
5. Notice that the question ('Upload anything from your computer") is never read, nor the help text.  Instead, it says "Drag file here or choose from folder.  No file chosen.  Select one or more files."

**Expected behavior**
The screen reader should read the question and the help text before saying "Drag file here..."

**Desktop (please complete the following information):**

- OS: MacOS
- Browser: Chrome
- Screen reader: VoiceOver

**Comment by rockycodes** [comment]
Draft PR has been started here, but tests are failing: https://github.com/civiform/civiform/pull/13854
